src/components/Icon/Arrow.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import { SVGProps } from "react";
+
+export interface ArrowIconProps extends SVGProps<SVGSVGElement> {
+  variant: "down" | "up" | "left" | "right"
+}
+
+const SvgUp = (props: ArrowIconProps) => {
+  let rotateDeg = undefined;
+  switch(props.variant) {
+    case "down":
+      rotateDeg = "0";
+      break;
+    case "left":
+      rotateDeg = "90";
+      break;
+    case "up":
+      rotateDeg = "180";
+      break;
+    case "right":
+      rotateDeg = "270";
+      break;
+  }
```

Comment:
Usually when we use a `switch` we would need a `default` case, also the directions seem like would be safer to have in an enum, what do you think of something along these lines:

```typescript
export interface ArrowIconProps {
  variant: Direction
}

export enum Direction {
  Up,
  Down,
  Left,
  Right,
}

const rotationDegree = {
  [Direction.Down]: "0",
  [Direction.Left]: "90",
  [Direction.Up]: "180",
  [Direction.Right]: "270",
}

// Usage

rotationDegree[props.variant]

```

Reply:
Nice!